src/app/js/formats/istex/Component.js

```diff
@@ -4,7 +4,7 @@ import { List } from 'material-ui/List';
 import translate from 'redux-polyglot/translate';
 import compose from 'recompose/compose';
 
-import FetchPaginatedDataHOC from '../../lib/FetchPaginatedDataHOC';
+import fetchPaginatedDataForComponent from '../../lib/fetchPaginatedDataForComponent';
```

Comment:
and if I call it fetchPaginatedData it would seem to be a service to fetch paginated data, which it is not.